uber/config.py

```diff
@@ -486,6 +522,7 @@ def _unrepr(d):
         c.PRICE_LIMITS[int(_opt)] = _val
     else:
         c.PRICE_BUMPS[date] = _val
+c.ORDERED_PRICE_LIMITS = sorted([val for key, val in c.PRICE_LIMITS.items()])
```

Comment:
Side note, you can do this:
```python
c.ORDERED_PRICE_LIMITS = sorted(c.PRICE_LIMITS.values())
```

Also, would it be better to store the keys here? Then instead of doing:
```python
for key, val in c.PRICE_LIMITS.items():
    if c.ORDERED_PRICE_LIMITS[current_price_tier+1] == val:
        difference = key - c.BADGES_SOLD
```

You could just do:
```python
difference = c.ORDERED_PRICE_LIMITS[current_price_tier + 1] - c.BADGES_SOLD
```

Would that work?